src/components/NewCommentForm.tsx

```diff
@@ -59,13 +64,15 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
 
     setSubmitting(true);
 
-    // it is very easy to forget about `await` keyword
-    await onSubmit({ name, email, body });
+    await dispatch(apiAddComment({
```

Comment:
The dispatch method is not handled in a try/catch block which could lead to unhandled promise rejection if there's an error.